File: Civitas-Java/Civitas/src/main/java/juegoTexto/JuegoTexto.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package juegoTexto;

import civitas.CivitasJuego;
import civitas.Dado;
import java.util.ArrayList;

/**
 * @author elena
 * @date 25/10/2019
 */
public class JuegoTexto {

    /**
     * @param args the command line arguments
     */
    public static void main(String[] args) {
        VistaTextual mivista = new VistaTextual();
        
        ArrayList<String> nombres = new ArrayList();
        nombres.add("Elena");
        nombres.add("Pepe");
        nombres.add("Juan");
        nombres.add("Laura");
        
        CivitasJuego mijuego = new CivitasJuego(nombres);
        
        Dado dado = Dado.getInstance();
        //Modificada la visibilidad del debug del dado
        dado.setDebug(true);
        
        Controlador con = new Controlador(mijuego, mivista);
        
        con.juega();
    }
    
}

File: Civitas-Ruby/lib/JuegoTexto.rb
# To change this license header, choose License Headers in Project Properties.
# To change this template file, choose Tools | Templates
# and open the template in the editor.

require_relative 'civitas_juego'
require_relative 'dado'
require_relative 'controlador'
require_relative 'vista_textual'

module Civitas
  class JuegoTexto
        
    def self.main
      mivista = Vista_textual.new
      
      @nombres = Array.new
      @nombres << 'Elena'
      @nombres << 'Pepe'
      @nombres << 'Laura'
      @nombres << 'Juan'      
      
      @juego = CivitasJuego.new(@nombres)
      @dado = Dado.instance      
      @dado.setdebug(true)
      
      @con = Controlador.new(@juego, mivista)
      @con.juega
    end
  end
  
  JuegoTexto.main
end
